metrics/public/public.go

```diff
@@ -26,6 +26,14 @@ var (
 		Namespace: "smh",
 		Name:      "version",
 	}, []string{"version"})
+	CurrentBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "smh",
+		Name:      "beacon",
+	}, []string{"curr_beacon"})
+	NextBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "smh",
+		Name:      "next_beacon",
+	}, []string{"beacon"})
```

Comment:
How about
```suggestion
	CurrentBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
		Namespace: "smh",
		Subset:    "beacon",
		Name:      "current,
	}, []string{"beacon"})
	NextBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
		Namespace: "smh",
		Subset:    "beacon",
		Name:      "next",
	}, []string{"beacon"})
```

to get metrics:
- `smh_beacon_current{beacon=}`
- `smh_beacon_next{beacon=}`